tests/samples/VarDefine.h

```diff
@@ -0,0 +1,13 @@
+extern int a;
+#define A a
+
+#define B b // ignored because there is no such variable
+
+extern int c;
+#define C c
+#undef C // removed
+
+extern float f;
```

Comment:
Why is there no variable generated for `f`?

Reply:
Because this commit is about defines
Define for `f` is private `__PRIVATE` and therefore filtered